scripts/convert_to_db_dump.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -eu
+
+TARFILE=$1
+DUMPFILE=$(basename $TARFILE).dump
+
+[[ -n $DATABASE_PASSWORD ]]  || (echo "DATABASE_PASSWORD not set" && exit 1)
+[[ -n $DB_BACKUP_DIR ]]      || (echo "DB_BACKUP_DIR not set" && exit 1)
+[[ -n $ENV ]]                || (echo "ENV not set" && exit 1)
+
+echo "Unpacking ..."
+
+tar xfz $TARFILE
+
+echo "Launching backup db ..."
+
+docker run --rm --name backup-db -d -v "$PWD/db_data:/var/lib/postgresql/data" -p 9432:5432 postgres:12.1-alpine
```

Comment:
This makes the backup db briefly available on port 9432 (with a strong password of course), until the backup job is complete. We probably avoid this if we ran the pg_dump in a container as well and had them just talk on the container network.

Reply:
Fixed, just couldn't live with it 🤣